geniza/corpus/models.py

```diff
@@ -1,29 +1,41 @@
 from django.db import models
 
 
-class Library(models.Model):
-    '''Library or archive that holds Geniza fragments'''
-    name = models.CharField(max_length=255, unique=True)
+class Collection(models.Model):
+    '''Collection at a library that holds Geniza fragments'''
+    library = models.CharField(max_length=255)
     abbrev = models.CharField('Abbreviation', max_length=255, unique=True)
-    url = models.URLField('URL', blank=True)
+    collection = models.CharField(
+        max_length=255, blank=True,
+        help_text='Collection name, if different than Library')
+    location = models.CharField(
+        max_length=255, help_text='Current location of the collection')
```

Comment:
idk what we're using this for -  if it's just display, `CharField` seems fine, but if we want to do anything with the actual info later we might wanna store real coords or geonames or something

Reply:
I thought about that too, and mentioned briefly in the working meeting when we discussed these changes; but I'm recommending we hold off on that until we have a use case, since our priority is getting the spreadsheet data migrated into the database.